.github/workflows/ci.yml

```diff
@@ -55,19 +55,36 @@ jobs:
           toolchain: ${{ matrix.toolchain_nightly }}
           default: false
 
+      - name: '`cargo test`'
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --verbose
+        env:
+          CARGO_UDEPS_TEST_TOOLCHAIN: ${{ matrix.toolchain_nightly }}
+
+      # `locked`, `dev-deps`
       - name: '`cargo build`'
         uses: actions-rs/cargo@v1
         with:
           command: build
           args: --verbose
 
-      - name: '`cargo test`'
+      # `locked`, `avoid-dev-deps`
+      - name: '`cargo install --debug --locked`'
         uses: actions-rs/cargo@v1
         with:
-          command: test
-          args: --verbose
-        env:
-          CARGO_UDEPS_TEST_TOOLCHAIN: ${{ matrix.toolchain_nightly }}
+          command: install
+          args: --path . --debug --verbose --locked
+
+      # https://github.com/rust-lang-nursery/failure/issues/342
```

Comment:
Can you remove this comment and activate the section below?

Reply:
Ok. I activated the section.